.env.example

```
@@ -7,3 +7,8 @@ GOOGLE_CLIENT_ID=your_google_client_id_here
 # Frontend Secrets (place in frontend/.env)
 VITE_GOOGLE_CLIENT_ID=your_google_client_id_here
 VITE_BASE_URL=http://localhost:1313
+
+# Optional WebRTC TURN relay (recommended for reliable calls across restrictive networks)
+VITE_WEBRTC_TURN_URL=turn:your-turn-server.example.com:3478
+VITE_WEBRTC_TURN_USERNAME=your_turn_username
+VITE_WEBRTC_TURN_CREDENTIAL=your_turn_credential
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_WEBRTC_TURN_CREDENTIAL key should go before the VITE_WEBRTC_TURN_URL key

(UnorderedKey)